Documentation/ReleaseNotes.md

```diff
@@ -7,250 +7,24 @@
 
 ## What's new
```

Comment:
Should we add the two PRs just merged?

Reply:
they are small fixes, like the two sample updates. i think we are fine without them.

they _will_ show up in the changelog (via the link contained in that file)